examples/react/view-transitions/src/routes/posts.route.tsx

```
@@ -1,4 +1,4 @@
-import { createFileRoute } from '@tanstack/react-router'
+import { createFileRoute, useRouter } from '@tanstack/react-router'
```

ESLint

[error] '@tanstack/react-router' imported multiple times.

(import/no-duplicates)